python/hail/api2/table.py

```diff
@@ -1967,5 +1967,33 @@ def explode(self, field):
 
         return Table(self._jt.explode(fields_rev[field]))
 
+    @property
+    @handle_py4j
+    def globals(self):
+        """Returns a struct expression including all global fields.
```

Comment:
add a test for tables also?

Reply:
will remove the table ones like we talked about